types/howler/index.d.ts

```diff
@@ -110,6 +110,7 @@ interface Howl {
     state(): 'unloaded' | 'loading' | 'loaded';
     load(): void;
     unload(): void;
+    stereo(): number | null;
```

Comment:
the documentation you provided does not indicate that `stereo` can be called with no parameters.

Reply:
Yes, their documentation is incorrect. Stereo should be documented the same as seek, rate etc in order for it to provide the "get" functionality they specify.

I will log an issue with them once I have been able to test which other methods also require document changes, but the stereo function is what I am using and have tested at this moment in time.

Reply:
Stereo cannot be passed without parameters. Here is the implementation:
https://github.com/goldfire/howler.js/blob/master/src/plugins/howler.spatial.js#L159

Reply:
Well, it is 'technically' possible, though the function call will just return the Howl's current stereo value.